docs/UserGuide.md

```diff
@@ -523,9 +620,9 @@ TutorAid student data are saved as a JSON file `[JAR file location]/data/tutorAi
 
 ***
 
-# 8. Command summary
+# 8. Command Summary
 
-### 8.1 Student commands
+### 8.1 Student Commands
 
 |---
 Action | Format and Examples
```

Comment:
For line 630 (Add student), I think it is better to remove the `...` behind `[pp/PARENT_PHONE]` in `add -s sn/STUDENT_NAME [sp/STUDENT_PHONE] [pn/PARENT_NAME] [pp/PARENT_PHONE]…​`